packages/web/src/scss/components/Collapse/_Collapse.scss

```
@@ -23,6 +23,11 @@
         overflow: visible;
     }
 
+    /* When it should be as a hidden inline element */
+    &:is(span) {
+        display: none;
```

Actually, it's the other way round 🙂. Inline elements need to be switched into the block-level mode and then it works:

```suggestion
        display: block;
```

---

@adamkudrna But I don't want it to be a block element. I, on the other hand, need it to remain `inline'. With this suggestion of yours, the element behaves differently and not as I expect. Try it yourself on the example with `data-more`, that after splitting, the content is inserted as if on a new line, but this was all intended to be inserted instead of the button and on the same line.